### ISSUE TITLE
Add Azure Arc | Windows Server ESU licenses utilization in Advisor Cost Workbook

### ISSUE THREAD

**Issue by alexmiloev** [microsoft/finops-toolkit#1178]
## 📝 Scenario
As a FinOps practitioner, I need to be able to see the Arc ESU license utilization in order to identify cost optimization opportunities 

## 💎 Solution
Identify the graph query portal is using here and add it to the workbook
[https://portal.azure.com/?feature.msaljs=true#view/Microsoft_Azure_ArcCenterUX/ArcCenterMenuBlade/~/usageOverview](url)
![Image](https://github.com/user-attachments/assets/1eb0184f-0164-4d53-ba89-24bcbea73c81)

We could use your help:
1. Please vote this issue up (👍) to prioritize it.
2. Leave comments to help us solidify the vision.

**Comment by arthurclares** [comment]
@alexmiloev  Do you happen to have an environment where we can test this?

**Comment by alexmiloev** [comment]
Yes, I can test a workbook in an environment with many ESU licenses and send you a sanitized screenshot.
